fe/fe-core/src/main/java/com/starrocks/fs/hdfs/HdfsFsManager.java

```diff
@@ -1214,11 +1214,11 @@ public List<TBrokerFileStatus> listPath(String path, boolean fileNameOnly, Map<S
                 resultFileStatus.add(brokerFileStatus);
             }
         } catch (FileNotFoundException e) {
-            LOG.info("file not found: " + e.getMessage());
-            throw new UserException("file not found: " + e.getMessage());
+            LOG.info("file not found: " + path, e);
```

Comment:
If these message will be returned to client, why log it here?

Reply:
```
LOG.info("file not found: {}", path, e);
```

Reply:
The returned message is different, it will be simplified to make it more understandable. 
So logging the detailed error here to help diagnose error.